tutorials/multiLayerPBF/constant/transportProperties

```diff
@@ -29,8 +29,8 @@ liquid
 
 powder
 {
-    kappa   (-0.07707   0.00075   0.0);
-    Cp      (747.568    0.0       0.0);
+    kappa   (0.21767  0.00102   0.0);
```

Comment:
Should we provide a reference for these properties?

Reply:
I estimated them from here: https://www.mdpi.com/2504-4494/6/1/2
We should probably have a properties database in a future release with citations. I think citing all properties might make the inputs hard to read.